dev/devicelab/bin/tasks/commands_test.dart

```diff
@@ -59,10 +65,18 @@ void main() {
       print('test: pressing "P" again...');
       run.stdin.write('P');
       await drive('none');
+      print('test: pressing "r" to perform a hot reload...');
+      run.stdin.write('r');
+      await drive('none');
```

Comment:
you may have to await the hot reload having completed (maybe by reading the output from stdout until you get notified, that'd be the most useful way in terms of testing things) otherwise it's possible the drive will race and connect before the hot reload has started.